portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACNavigation.path

```diff
@@ -27,6 +27,26 @@
 	<td></td>
 </tr>
 <!-- TABS/OPTIONS AVAILABLE IN DIFFERENT PLACES E.G: ASSETS, SEGMENTS, SITES -->
+<tr>
+	<td>DEFINITIONS_ATRIBUTES_TABS</td>
+	<td>//ul[contains(@class,'nav-root')]//b[text()='${key_tabName}']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>DEFINITIONS_ENTITY_NAME</td>
+	<td>//li[contains(@class, 'list-group-item')]//a[text()='${key_entityName}']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ENTRIES_LIST_DROPDOWN_OPTION</td>
+	<td>//div[contains(@class, 'dropdown-menu show')]//a[text()='${key_dropdownOption}']</td>
```

Comment:
Hi @91nilo , when using `contains`, please try to not use multiple options, in this case multiple classes, since it will let your code more fragile and susceptible to break for a change on the front-end.

So, I would suggest you to use something like:
`<td>//div[contains(@class, 'dropdown-menu') and contains (@class,'show')]//a[text()='${key_dropdownOption}']</td>`

or try to see if you are able to create a path with a single class element (the best way).

So, if the order of the classes changes, your path will not break.